<ui version="4.0" >
 <class>AbstractItemView</class>
 <widget class="QWidget" name="AbstractItemView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>670</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <layout class="QVBoxLayout" name="_1" >
     <item>
      <layout class="QHBoxLayout" name="_2" >
       <item>
        <layout class="QVBoxLayout" name="_4" >
         <item>
          <widget class="QLineEdit" name="filterEdit" />
         </item>
         <item>
          <widget class="QTableView" name="tableView" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>500</height>
            </size>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled" >
            <bool>true</bool>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QPushButton" name="newButton" >
           <property name="text" >
            <string>&amp;Novo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewButton" >
           <property name="text" >
            <string>Ve&amp;r</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editButton" >
           <property name="text" >
            <string>E&amp;ditar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton" >
           <property name="text" >
            <string>De&amp;letar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="printButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>&amp;Relat√≥rio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton" >
           <property name="text" >
            <string>&amp;Fechar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
